src/components/Markdown.jsx

```diff
@@ -365,9 +365,7 @@ const useStyles = makeStyles(theme => ({
   },
 }))
 
-let previousHeadings = []
-
-const HeadingLevelToComponent = (level, props) => {
+const HeadingLevelToComponent = (level, props, previousHeadings) => {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Arrow function has a complexity of 7. (complexity)](https://app.codacy.com/app/davwheat/GCSE-Revision-Website/pullRequest?prid=4583220)